applications/CompressiblePotentialFlowApplication/python_scripts/compute_lift_process.py

```diff
@@ -14,46 +29,98 @@ def __init__(self, Model, settings ):
             "model_part_name": "please specify the model part that contains the surface nodes",
             "velocity_infinity": [1.0,0.0,0.0],
             "reference_area": 1.0,
+            "moment_reference_point" : [0.0,0.0,0.0],
             "create_output_file": false
         }''')
 
         settings.ValidateAndAssignDefaults(default_parameters)
 
         self.body_model_part = Model[settings["model_part_name"].GetString()]
-        self.velocity_infinity = [0,0,0]
-        self.velocity_infinity[0] = settings["velocity_infinity"][0].GetDouble()
-        self.velocity_infinity[1] = settings["velocity_infinity"][1].GetDouble()
-        self.velocity_infinity[2] = settings["velocity_infinity"][2].GetDouble()
+        self.fluid_model_part = self.body_model_part.GetRootModelPart()
         self.reference_area =  settings["reference_area"].GetDouble()
         self.create_output_file = settings["create_output_file"].GetBool()
+        self.moment_reference_point = KratosMultiphysics.Vector(3)
+        self.moment_reference_point = settings["moment_reference_point"].GetVector()
+        
 
     def ExecuteFinalizeSolutionStep(self):
         KratosMultiphysics.Logger.PrintInfo('ComputeLiftProcess','COMPUTE LIFT')
 
-        rx = 0.0
-        ry = 0.0
-        rz = 0.0
+        total_force = KratosMultiphysics.Vector(3)
+        total_moment = KratosMultiphysics.Vector(3)
 
         for cond in self.body_model_part.Conditions:
             n = cond.GetValue(KratosMultiphysics.NORMAL)
             cp = cond.GetValue(KratosMultiphysics.PRESSURE)
 
-            rx += n[0]*cp
-            ry += n[1]*cp
-            rz += n[2]*cp
+            # Computing forces
+            total_force[0] += n[0]*cp
+            total_force[1] += n[1]*cp
+            total_force[2] += n[2]*cp
+
+            # Computing moment
+            mid_point = cond.GetGeometry().Center()
+            lever = mid_point-self.moment_reference_point
+            total_moment += CrossProduct(lever, n*(-cp))
+
+        Cf = KratosMultiphysics.Vector(3)
```

Comment:
```suggestion
        lift_coefficient = total_force / self.reference_area
```

Reply:
This is not the lift coefficient but the force coefficient. I will change the name to make it easier to understand.

Reply:
then you got the point ;)